sara_arm_joint_publisher:
  type: "joint_state_controller/JointStateController"
  publish_rate: 50

sara_arm_controller:
  type: "velocity_controllers/JointTrajectoryController"
  publish_rate: 50               # default: 50
  joints:
    - torso_arm_joint
    - arm_shoulder_joint
    - arm_rot_elbow_joint
    - arm_elbow_joint
    - arm_rot_wrist_joint
    - arm_high_low_wrist_joint
    - arm_low_wrist_grasp_socket_joint


  gains: # Required because we're controlling a velocity interface
    torso_arm_joint:                  {p: 100,  d: 1, i: 1, i_clamp: 1}
    arm_shoulder_joint:               {p: 100,  d: 1, i: 1, i_clamp: 1}
    arm_rot_elbow_joint:              {p: 100,  d: 1, i: 1, i_clamp: 1}
    arm_elbow_joint:                  {p: 100,  d: 1, i: 1, i_clamp: 1}
    arm_rot_wrist_joint:              {p: 100,  d: 1, i: 1, i_clamp: 1}
    arm_high_low_wrist_joint:         {p: 100,  d: 1, i: 1, i_clamp: 1}
    arm_low_wrist_grasp_socket_joint: {p: 100,  d: 1, i: 1, i_clamp: 1}


  # Velocity commands timeout [s], default 0.5
  cmd_vel_timeout: 0.5
